src/read_csv.py

```diff
@@ -2,17 +2,19 @@
 import ast
 
 
-def read_swaps_from_csv(filename, read_swaps_splitted=False):
+def read_swaps_from_csv(filename, read_swaps_splitted=False, assume_only_halve_of_trades_from_uniswap_is_migrating=False):
     with open(filename, newline='') as f:
         reader = csv.reader(f)
         data = list(reader)
         orders = dict()
         first = True
-        for row in data:
+        for cnt, row in enumerate(data):
             # Skip header.
             if first:
                 first = False
                 continue
+            if assume_only_halve_of_trades_from_uniswap_is_migrating and cnt % 2 == 0:
```

Comment:
Perhaps it is better to use a `random.sample()` of the rows in the end of the loop, or in alternative shuffle data before the start of the loop. Probably won't make a big difference, but I guess we shouldn't assume any kind of deterministic behaviour of how the half of the users that migrate is formed.